classes/requests/helpers/class-pb2b-order-lines.php

```diff
@@ -88,7 +88,7 @@ public static function get_product_name( $order_item ) {
 	 */
 	public static function get_line_unit_price( $order_item ) {
 		$quantity      = 0 === abs( $order_item->get_quantity() ) ? 1 : abs( $order_item->get_quantity() );
-		$item_subtotal = round( ( $order_item->get_total() ) / $quantity * 100, 2 );
+		$item_subtotal = abs( round( ( $order_item->get_total() ) / $quantity * 100, 2 ) );
```

Comment:
@montazarr Why was this needed? Just so that i know

Reply:
@MichaelBengtsson the new endpoint expects a positive number. It fails otherwise.